enterprise/api/v1/views/enterprise_customer_sso_configuration.py

```diff
@@ -146,6 +146,10 @@ def oauth_orchestration_complete(self, request, configuration_uuid, *args, **kwa
                 ' not been marked as submitted.'
             )
 
+        # Mark the configuration record as active IFF this the record has never been configured.
+        if sso_configuration_record.configured_at:
```

Comment:
Logic to match comment
```suggestion
        # Mark the configuration record as active IFF this the record has never been configured.
        if not sso_configuration_record.configured_at:
```